spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewritePositionDeleteFilesAction.java

```diff
@@ -739,23 +749,31 @@ private Table createTablePartitioned(int partitions, int files, int numRecords)
   }
 
   private Table createTableUnpartitioned(int files, int numRecords) {
+    return createTableUnpartitioned(files, numRecords, 2);
```

Comment:
I would probably just keep a 1 off table creator for V3 Tables rather than adding a new method which is parameterized solely for a single usage. 

So we just inline 

```java

    Table table =
        validationCatalog.createTable(
            TableIdentifier.of("default", TABLE_NAME),
            SCHEMA,
            PartitionSpec.unpartitioned(),
            tableProperties());
            tableProperties(3));

    writeRecords(table, files, numRecords);
```

In the V3 test

Reply:
makes sense to me, I've updated that